error_notify/total_override.py

```diff
@@ -0,0 +1,11 @@
+import os, sys
+
+def placeholder_notific_fn():
+	print("PLACEHOLDER CALLED")
+
+def Notify(type_, value, traceback):
```

Comment:
can we name it something else, `Notify` makes it confusing with the upcoming notification module

Reply:
Done